### ISSUE TITLE
Google Cloud Build & Deploy Process

### ISSUE THREAD

**Issue by MorpheusZero** [OpenCardsOnline/opencardsonline#6]
**Is your feature request related to a problem? Please describe.**
When code is merged into different branches, nothing actually happens right now.

**Describe the solution you'd like**
I am proposing that there should be 2 main environments for the application.

   - development: This is where code will go when it has been merged into the main dev branch in the repo. It will cause a build and deploy the code to our `dev.opencardsonline.com` instance.
   - production: This is where code will go when it has been merged to the main MASTER branch in the repo. It will cause a build and deploy the code to our `app.opencardsonline.com` instance.

A few things need to happen here:

   - [ ] DOCKERFILE Issue has to be completed first.
   - [ ] The two main branches (master and dev) need to exist in the repo.
   - [ ] A configuration build file YAML needs to be created for the Google Cloud Build process.
   - [ ] Finally, merging code into a branch needs to kick off the automated processes and deployments need to be successful.

**Describe alternatives you've considered**

**Additional context**
